app/controllers/events_controller.rb

```diff
@@ -17,6 +17,10 @@ class EventsController < ApplicationController
     render json: {error: 'Authorization error'}, status: :unauthorized
   end
 
+  rescue_from GoogleEvent::EventInTimeSpanError do
+    render json: {conference_room_id: ['Another event in time span for chosen room']}, status: :unprocessable_entity
```

Comment:
How about reusing the `EventInTimeSpanError` error message instead of creating a custom one?